### PR TITLE
[Docs] PR for Update documentation for NTP Enhancements

### PR DESCRIPTION
**Pull request by lizayugabyte**
Addresses https://yugabyte.atlassian.net/browse/PLAT-3012

Preview: https://deploy-preview-12474--infallible-bardeen-164bc9.netlify.app/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/aws/

### CODE REVIEW

**Comment by lizayugabyte** [comment]
@hkandala When do you think you'd be able to review the PR ?  (this doc needs to be published ASAP)

**Comment by hkandala** [comment]
LGTM, I would recommend to wait for review from Rahul Prabhu too.
@favre49 please help reviewing this doc. Thanks!

**Review by hkandala — APPROVED** [review]

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/azure.md
Comment (favre49) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/azure.md:101]:
```suggestion
  - Select **Use provider’s NTP server** to enable cluster nodes to connect to the Azure internal time servers. For more information, consult the Microsoft Azure documentation such as [Time sync for Linux VMs in Azure](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/time-sync). 
```

**Review by favre49 — APPROVED** [review]
LGTM, one minor comment. 

Do you think we should update the pictures again? Since you took them two things have changed - there is now a beta label on the NTP setup, and the default option is "Use Provider's NTP server" for GCP/AWS/Azure, which will be in 2.13.2 as well. I'm not sure whether this matters enough though, it is nitpicky.

**Pull request merged**
